spring-cloud-gcp-samples/spring-cloud-gcp-vision-api-sample/src/main/resources/static/index.html

```diff
@@ -5,10 +5,22 @@
     <title>Google Cloud Vision API with Spring code sample</title>
 </head>
 <body>
+
 <div>
-    <form action="/vision">
+    <h1>Image Label Annotations</h1>
+    <p>Returns labels classifying the content of the image:</p>
+    <form action="/extractLabels">
+        URL of image to be uploaded: <input type="text" name="imageUrl"/> <input type="submit"/>
```

Comment:
Can we pre-fill or suggest using the images already on the classpath?

Reply:
So the intention here was not to have users upload images from disk but to specify the URL of an image on the web; I figured this would be much more convenient than uploading/including things on classpath.

I've reworded the description to say "Web URL of image to analyze" to clarify the intention.